pkg/proxy/util/utils.go

```diff
@@ -52,6 +52,18 @@ func IsZeroCIDR(cidr string) bool {
 	return false
 }
 
+// IsZeroMask returns true if the ip mask is zero.
+func IsZeroMask(ipNet *net.IPNet) bool {
```

Comment:
`IsZeroMask` is not a totally obvious name.

How about if you replace `func IsZeroCIDR(cidr string)` with `func IsZeroCIDR(cidr *net.IPNet)` ? (The existing function is only used from one place, and that place could use a `*net.IPNet` version just as easily as a `string` version.) (You'll need to update its unit test too.)